app/lib/methods/helpers/mergeSubscriptionsRooms.ts

```diff
@@ -18,7 +25,7 @@ export const merge = (
 	if (room) {
 		room = EJSON.fromJSONValue(room) as ISubscription;
 		if (room._updatedAt) {
-			subscription.lastMessage = normalizeMessage(room.lastMessage);
+			subscription.lastMessage = normalizeMessage(room.lastMessage) as IMessage;
```

Comment:
This here is related to putting null in the return of that function.

Reply:
Reason previously explained

Reply:
But if it's null, we shouldn't cast to IMessage.
`subscription.lastMessage` can be null.
For example, when you create a room, it has no messages.